### ISSUE TITLE
Test installed astropy with pytest --pyargs astropy

### ISSUE THREAD

**Issue by astrofrog** [astropy/astropy#10411]
As illustrated with https://github.com/astropy/astropy/issues/10410, we should make it so one of our CI configurations actually ignores the ``setup.cfg`` otherwise issues like https://github.com/astropy/astropy/issues/10410 and https://github.com/astropy/astropy/pull/10383 will creep in.

@Cadair - any suggestions on how we could do this as an option in tox?

**Comment by Cadair** [comment]
change directory before running `pytest`?

**Comment by astrofrog** [comment]
We already do change to a temporary directory but it searches for setup.cfg in any ancestor directory.

**Comment by Cadair** [comment]
Ah, well can always change out to `/` or something?

**Comment by astrofrog** [comment]
Yeah - just to be clear to anyone reading this, this would just be an additional factor in the env name which would cause this, most environments should just be run as they are now.

**Comment by mhvk** [comment]
Note that there are two runs on travis that do not use `tox` (I wanted something that ensures things keep working on Debian without it and (ab)used the big-endian and arm runs, sorry...), so if it is tricky within `tox`, they might be logical candidates - I think all it takes is generating the environment one directory up (assuming travis allows that).

https://github.com/astropy/astropy/blob/499195018d294f7fae9bc2f658475c00065dda49/.travis.yml#L254-L263
